native/src/main/java/io/ballerina/stdlib/websocket/WebSocketResourceDispatcher.java

```diff
@@ -414,6 +416,7 @@ public static void dispatchOnText(WebSocketConnectionInfo connectionInfo, WebSoc
                 webSocketConnection.readNextFrame();
                 return;
             }
+            boolean validationEnabled = (boolean) wsService.getBalService().getNativeData("validation");
```

Comment:
We can use the defined constant

Reply:
Fixed with [4074808](https://github.com/ballerina-platform/module-ballerina-websocket/pull/688/commits/40748087e4cd31fb11334876065e18315f674b75)